### DISCUSSION TITLE
New BIOS for GA503RW, RM 318, 319. Safe to update

### DISCUSSION THREAD

**Discussion by Zylon676** [seerge/g-helper#3122] [General]
Wanted to know if there any issues in updating to the BIOS wait for tests and such to double check.

**Comment by seerge** [comment]
@Zylon676 i don't know :) may be try to update and see if there are any improvements ? As I remember all G15 were suffering from a strange fan behaviors (like asus has accidentally swapped GPU and CPU fans?) in older BIOSes

**Reply by Zylon676** [reply]
It says optional so I won't be doing so.

**Comment by badrmv** [comment]
I have just updated it, ofc no noticeable difference for now but some people have said that sleep mode works better now, I won't be testing this actually as I don't use sleep mode anyway but if there is any significant improvements that are worth mentioning or any mistakes this update brings I will comment them later, if I didn't update this comment it's not worth updating the bios to 319 do it or not nothing really changes

**Comment by victorandre957** [comment]
I updated 2 hours ago and then started playing, I noticed that the GPU temperature was a little lower, yesterday playing the same game (lies of p) in the same conditions it was at 88/89° C today it was in the 80 range along with the CPU. Has anyone else noticed this?
